File: Bai_28/main.cpp
#include<iostream>
#include<vector>
using namespace std;

bool KTSNT(int a)
{
    if(a <= 1)
        return false;
    for(int i = 2; i < a; i++)
        if(a % i == 0)
            return false;
    return true;
}

int main()
{
    int N;
    cout << "Nhap so phan tu SNT can coi: ";
    cin >> N;
    vector<int> list;
    int count(2);
    while (list.size() < N)
    {
        if(KTSNT(count))
            list.push_back(count);
        count += 1;
    }
    
    for(int i : list)
        cout << i << ' ';

    return 0;
}

File: Bai_28/main.py
def KTSNT(a):
    if a <= 1:
        return False
    for i in range(2,a):
        if a % i == 0:
            return False
    return True
    
N = int(input("Nhập số phần tử SNT cần biết: "))
list= []
count = 2
while len(list) < N:
    if KTSNT(count):
        list.append(count)
    count += 1
        
print(list)    